source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql

```diff
@@ -69,71 +69,85 @@ BEGIN
-			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(InPrisonMoreThan5YearsAgo)				AS PrisonMoreThan5YearsAgo,
-			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(CurrentlyInPrisonOrInPrisonWhenFirstSeen) AS CurrentlyInPrisonOrWhenFirstSeen,
-			rn.TreatmentPhec																				AS TreatmentRegion,
-			rn.AlcoholMisuse																				AS AlcoholUse,
+			ethnicity.Label																					AS EthnicGroup,
+			patient.UkBorn																					AS UKBorn,
+			UPPER(country.Name)																				AS BirthCountry,
+			patient.YearOfUkEntry																			AS UkEntryYear,
+			CONVERT(varchar, clinicalDetails.SymptomStartDate, 103)											AS SymptomOnset,
+			CONVERT(varchar, clinicalDetails.FirstPresentationDate, 103)									AS DatePresented,
+			CONVERT(varchar, clinicalDetails.DiagnosisDate, 103)											AS DateOfDiagnosis,
+			CONVERT(varchar, clinicalDetails.TreatmentStartDate, 103)										AS StartOfTreatment,
+			drugs.Status																					AS DrugUse,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(drugs.InPastFiveYears)					AS DrugMisuseLast5Years,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(drugs.MoreThanFiveYearsAgo)				AS DrugUseMoreThan5YearsAgo,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(drugs.IsCurrent)							AS CurrentDrugUse,
+			homeless.Status																					AS Homeless,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(homeless.InPastFiveYears)					AS HomelessLast5Years,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(homeless.MoreThanFiveYearsAgo)			AS HomelessMoreThan5YearsAgo,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(homeless.IsCurrent)						AS CurrentlyHomeless,
+			prison.Status																					AS Prison,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(prison.InPastFiveYears)					AS InPrisonInLast5Years,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(prison.MoreThanFiveYearsAgo)				AS PrisonMoreThan5YearsAgo,
+			dbo.ufnGetFormattedSiteDiseaseDurationStatusForForest(prison.IsCurrent)							AS CurrentlyInPrisonOrWhenFirstSeen,
+			servicePhec.Name																				AS TreatmentRegion,
+			socialRiskFactors.AlcoholMisuseStatus															AS AlcoholUse,
 			lab.[SpecimenTypeCode]																			AS SmearSample,
 			'Positive'																						AS SmearSampleResult,
 			lab.SpecimenDate																				AS SpecimenDate,
 			lab.ReferenceLaboratoryNumber																	AS ReferenceLaboratoryNumber,
 			GETUTCDATE()																					AS ExtractDate,
 			patient.LocalPatientId																			AS LocalPatientId,
-			Age																								AS Age,
-			--hospital.CaseManagerUsername																	AS OwnerUserId,
-			CaseManager																						AS CaseManager,
-			rn.Consultant																					AS PatientsConsultant,
+			(0 + Convert(Char(8),GETDATE(),112) - Convert(Char(8),patient.Dob,112)) / 10000					AS Age,
```

Comment:
What is happening on this line?

Reply:
I think that's a very complicated way to work out the patient's age at time of notification - we have a function that does that, which can be called by passing in date of birth and notification date - https://github.com/publichealthengland/ntbs-reporting/blob/master/source/dbo/Functions/Reusable%20Notification/ufnGetAgefrom.sql

Reply:
Using the function now, I thought we might have one but forgot to look